superset/assets/javascripts/SqlLab/components/AceEditorWrapper.jsx

```diff
@@ -8,6 +8,22 @@ import { areArraysShallowEqual } from '../../reduxUtils';
 
 const langTools = ace.acequire('ace/ext/language_tools');
 
+const keywords = (
```

Comment:
oh it's just less characters to type :) .

I couldn't find how to import them from ace/brace so I copied their source code and they did it that way...